homeassistant/components/daikin/config_flow.py

```diff
@@ -22,24 +23,46 @@ class FlowHandler(config_entries.ConfigFlow):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
-    async def _create_entry(self, host, mac):
+    async def _create_entry(self, host, mac, key=None, uuid=None, password=None):
```

Comment:
This doesn't have to be a coroutine since we don't await inside. Just make it a callback.